File: app/src/main/java/com/example/energo_monitoring/checks/data/api/DeviceInfo.java
package com.example.energo_monitoring.checks.data.api;

public class DeviceInfo {
    private String deviceName;
    private String deviceNumber;
    private int typeId;
    private String lastCheckDate;

    public DeviceInfo(String deviceName, int typeId) {
        this.deviceName = deviceName;
        this.typeId = typeId;
    }

    public int getTypeId() {
        return typeId;
    }

    public void setTypeId(int typeId) {
        this.typeId = typeId;
    }

    public String getName() {
        return deviceName;
    }

    public void setName(String name) {
        this.deviceName = name;
    }

    public String getLastCheckDate() {
        return lastCheckDate;
    }

    public void setLastCheckDate(String lastCheckDate) {
        this.lastCheckDate = lastCheckDate;
    }

    public String getDeviceName() {
        return deviceName;
    }

    public void setDeviceName(String deviceName) {
        this.deviceName = deviceName;
    }

    public String getDeviceNumber() {
        return deviceNumber;
    }

    public void setDeviceNumber(String deviceNumber) {
        this.deviceNumber = deviceNumber;
    }

    public int getFilledState(){ return 0; };
}

File: app/src/main/java/com/example/energy_monitoring/checks/data/api/DeviceInfo.kt
package com.example.energy_monitoring.checks.data.api

import com.example.energy_monitoring.checks.data.devices.Field
import com.google.gson.annotations.SerializedName

open class DeviceInfo {
    @SerializedName("id2")
    open var id = 0
    var dataId = 0
    var deviceName = Field("")
    var deviceNumber = Field("")
    var typeId = 0
    var lastCheckDate = Field("")
}